resources/styles/global/loadable.less

```diff
@@ -0,0 +1,29 @@
+@-webkit-keyframes fadeIn { from { opacity: 0; } to { opacity: 1; } }
+@-moz-keyframes fadeIn { from { opacity: 0; } to { opacity: 1; } }
+@keyframes fadeIn { from { opacity: 0; } to { opacity: 1; } }
+
+.refresh-button-delayed {
```

Comment:
I did that so one button could have either delay or no delay via css.  I could consolidate the classnames better.